frontend/src/ui/components/CreateAssessment/SelectType.jsx

```diff
@@ -1,6 +1,9 @@
 import React, { useEffect, useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
 import styled from "styled-components";
+import axios from "../../../api/axios";
+
+
```

Comment:
why is this here? this belongs to the frontend section..

Reply:
it was something I was working on before, I forgot to remove it